oeps/oep-0014-proc-archive-repos.rst

```diff
@@ -77,19 +77,11 @@ Archive Steps
 
 These steps should be followed for all repos within the edX organization(forks included). After some experiments with keeping archived repos in the ``edx`` organization, we've learned that having abandoned code show up in searches hinders work to understand the current state of the system and the risk around new work, particularly deprecations and API changes. Thus we decided to move all archived repositories to a separate org.
 
-1. Update the README.rst file in the repository to state that it is archived, using the `README Archive Statement`_ below.
+1. (Recommended) Update the README.rst file in the repository to state that it is archived, using the `README Archive Statement`_ below.
```

Comment:
**Author Concern:** There was debate as to whether this is recommended and optional, or if we should require this step? I'd like to require it, but others (@feanil?) I think thought this should be optional or removed.

Reply:
Maybe we can remove the general "this is archived" statement requirement, but keep a recommendation here of leaving an explanation of why it was archived.

Reply:
What about this?

> 1. Update the README.rst file in the repository to add a brief note about why the repo is being archived, and what is serving as its replacement (where applicable). This may be as simple as a linking to the appropriate DEPR ticket.

(Basically just moving that text up from below.)

Reply:
Pushed a change, see what you think.